backend/app/tests/conftest.py

```
@@ -27,26 +28,33 @@ def override_dependencies():
 def client():
     with TestClient(app) as c:
         yield c
 
-
 @pytest.fixture(scope="function")
-def integration_client():
-    # Same app, just semantic distinction
-    with TestClient(app) as c:
-        yield c
+def integration_client(client):
+    yield client
```

Ruff (0.14.14)

No newline at end of file

Add trailing newline

(W292)